deployment/Chart.yaml

```diff
@@ -1,9 +1,9 @@
 apiVersion: v2
 name: dcgm-exporter
 description: A Helm chart for DCGM exporter
-version: "2.6.11"
+version: "3.0.0"
 kubeVersion: ">= 1.13.0-0"
```

Comment:
I think it's time to update this. No way we are still supporting 1.13 API

Reply:
This is updated to 1.19.